scss/_progress.scss

```diff
@@ -26,80 +26,49 @@
   // Reset the default appearance
   appearance: none;
   // Set overall border radius
-  @include border-radius($border-radius);
+  @include border-radius($progress-border-radius);
 }
 .progress[value]::-ms-fill {
-  background-color: $progress-bar-color;
   // Remove right-hand border of value bar from IE10+/Edge
   border: 0;
 }
 .progress[value]::-webkit-progress-bar {
   background-color: $progress-bg;
-  @include border-radius($border-radius);
+  @include border-radius($progress-border-radius);
```

Comment:
Expected item on line 37 to appear before line 36. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets